src/components/filter/row/MultipleStringFIlterRow.tsx

```diff
@@ -98,50 +146,58 @@ export default function MultipleStringFilterRow({ config }: MultipleStringFilter
 			)}
 			<div className={filterContentClassName} ref={rootRef}>
 				<div className={inputRootClassName} onClick={rootOnClick}>
-					{config.values.map((value, index) => (
-						<Bubble
-							key={index}
-							size='small'
-							removeIconType='white'
-							submitKeyCodes={[KeyCodes.TAB]}
-							className='filter__bubble'
-							value={value}
-							onSubmit={valueBubbleOnChangeFor(index)}
-							onRemove={valueBubbleOnRemoveFor(index)}
-							autocompleteVariants={config.autocompleteList}
-							isValid={
-								config.validateBubbles
-									? config.autocompleteList
-										? config.autocompleteList.includes(value.trim())
+					<div onKeyDown={focusBubble}>
+						{config.values.map((value, index) => (
+							<Bubble
+								ref={e => (bubble.current[index] = e)}
+								key={index}
+								id={index}
+								selectNext={index + 1}
```

Comment:
you can have focusBubble fn which accepts index
and then you can pass selectNext && selectPrev as
`selectNext={() => focusBubble(index + 1)}`